### PR TITLE
First section of Portfolio Sketlon Project:

### PR DESCRIPTION
**Pull request by nmeziany**
First section of Portfolio Sketlon Project:
-Added header(nav bar with logo).
-Added Presentation section.

### CODE REVIEW

File: REDEME.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    this project consist to set-up linters in github and create basic HTML page and style it!
+
+## Built With
+
+- HTML and CSS
+
+
+## Authors
+
+- GitHub: [@nmeziany]
```
Comment (elmar8287) [inline REDEME.md]: - [x] Please, consider that your README file name is incorrect - it is "REDEME". After fixing it, please, consider making it more informative - you can refer MICROVERSE template

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Skeleton</title>
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo"><h2>My logo</h2></div>
```
Comment (elmar8287) [inline index.html]: - [x] Please, consider that the logo has a link (a tag)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="menu"><img src="images/Enabled.png" alt="" /></div>
+      </nav>
+    </header>
+    <main>
+      <section class="hello">
+        <div class="hellosection">
+          <div class="box box1">I’m Mariane Glad to see you!</div>
+          <div class="box box2">
```
Comment (elmar8287) [inline index.html]: - [x] Please, use for the main title (e.g., "I'm Marianne ...") a `h1` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </div>
+          <div class="box box3">
+            <div>LET'S CONNECT</div>
+            <div>
```
Comment (elmar8287) [inline index.html]: - [x] Please, use for "LET`S CONNECT" subtitle a header tag `hx` (examples: h1, h2 ,h3, etc...)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    </header>
+    <main>
+      <section class="hello">
+        <div class="hellosection">
+          <div class="box box1">I’m Mariane Glad to see you!</div>
+          <div class="box box2">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
```
Comment (elmar8287) [inline index.html]: - [x] Please, use a `p` tag for the paragraph (e.g., "I'm a software ...")

File: index.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+          </div>
+          <div class="box box3">
+            <div>LET'S CONNECT</div>
+            <div>
+              <img class="social" src="images/twitter.svg" alt="" /><img class="social" src="images/Linkedin.svg" alt="" />
+              <img class="social" src="images/M.svg" alt="" /><img class="social" src="images/cat.svg" alt="" /> <img class="social" src="images/Y.svg" alt="" />
```
Comment (elmar8287) [inline index.html]: - [x] Please, consider the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @nmeziany ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [x] Please, consider comments under review
- [x] Please, fix your linter error
Screen Shot 2022-01-08 at 02 05 05


#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: REDEME.md (hunk 1 above)
Comment (nmeziany) [reply REDEME.md]: Modification for README file done and correction of its name too!

File: index.html (hunk 2 above)
Comment (nmeziany) [reply index.html]: Modifications done for the logo adding a link for it

File: index.html (hunk 3 above)
Comment (nmeziany) [reply index.html]: h1 tag added for the main title!

File: index.html (hunk 4 above)
Comment (nmeziany) [reply index.html]: h3 tag added for LET'S CONNECT subtitle

File: index.html (hunk 5 above)
Comment (nmeziany) [reply index.html]: Added p tag for the paragraph

File: index.html (hunk 6 above)
Comment (nmeziany) [reply index.html]: Created list for icons and link each icon to an a tag!

**Review by aliabbani — APPROVED** [review]
Hi @nmeziany  ✋, 

APPROVED 🟢🟢🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  🎉  🎉  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

[OPTIONAL]  **Please make your README file more descriptive - please use [this template ](https://github.com/microverseinc/readme-template)but remember to customize it to your project.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**